libs/designer-ui/src/lib/utils/utils.ts

```diff
@@ -341,8 +341,8 @@ export function getStatusString(status: string, hasRetries: boolean): string {
 
     case Constants.STATUS.TIMEDOUT:
       return intl.formatMessage({
-        defaultMessage: 'Timed Out',
-        id: 'MsCHhQ',
+        defaultMessage: 'Timedout',
```

Comment:
I think this should stay as two words
```suggestion
        defaultMessage: 'Timed out',
```

Reply:
Thanks for catching that! Just a typo on my side. 😅 Will fix.